sdk/go/common/workspace/plugins.go

```diff
@@ -802,14 +803,16 @@ func (spec PluginSpec) GetSource() (PluginSource, error) {
 				return newGithubSource(url, spec.Name, spec.Kind)
 			case "gitlab":
 				return newGitlabSource(url, spec.Name, spec.Kind)
+			case "http", "https":
+				return newHTTPSource(spec.Name, spec.Kind, url), nil
 			default:
-				return newPluginURLSource(spec.Name, spec.Kind, spec.PluginDownloadURL), nil
+				return nil, fmt.Errorf("unknown plugin source scheme %s", url.Scheme)
```

Comment:
nit: consider adding a separator between the "%s" and the message to differentiate the input from the message
```suggestion
				return nil, fmt.Errorf("unknown plugin source scheme: %s", url.Scheme)
```